ui/app/AppLayouts/Wallet/components/SendModalContent.qml

```diff
@@ -91,33 +91,37 @@ Item {
         }
     }
 
-    Input {
-        id: txtAmount
-        //% "Amount"
-        label: qsTrId("amount")
-        anchors.top: parent.top
-        //% "Enter amount..."
-        placeholderText: qsTrId("enter-amount...")
-        validationError: amountValidationError
-    }
-
-    AssetSelector {
-        id: selectAsset
-        assets: walletModel.assets
-        anchors.top: txtAmount.bottom
-        anchors.topMargin: Style.current.padding
-        anchors.right: parent.right
-        width: 86
-        height: 28
+    AssetAndAmountInput {
+      id: txtAmount
+      assets: walletModel.assets
+      defaultCurrency: walletModel.defaultCurrency
+      errorMessage: amountValidationError
+      anchors.top: parent.top
+      onInputChanged: {
+        if (isNaN(input)) {
+          return
+        }
+        if (input === "") {
+          txtAmount.fiatBalance = "0.00"
+        } else {
+          txtAmount.fiatBalance = walletModel.getFiatValue(input, assetSymbol, currency)
+        }
+      }
+      onFiatValueChanged: {
+        txtAmount.text = walletModel.getCryptoValue(input, currency, assetSymbol)
+      }
```

Comment:
As mentioned [https://github.com/status-im/nim-status-client/pull/703/commits/a8a597a71f255a0d247e18a05a41e1192b78f9e5#r466335566](here) I'm not super convinced by putting this here, but I also think it doesn't make too much of a difference at the moment.